From: James Ralston <james.d.ralston@intel.com>
Date: Fri, 8 Feb 2013 17:33:38 -0800
Subject: [PATCH] iTCO_wdt: Add Device IDs for Intel Wellsburg PCH

Git-commit: 6e6680e3effbf7cff444405a990dbc355dc3a96f
Patch-mainline: v3.9-rc1
References: fate#313702
Target: SLE-11 SP3

This patch adds the Watchdog Timer Device IDs for the Intel Wellsburg PCH

Signed-off-by: James Ralston <james.d.ralston@intel.com>
Acked-by: Peter Tyser <ptyser@xes-inc.com>
Signed-off-by: Samuel Ortiz <sameo@linux.intel.com>
Acked-by: Chun-Yi Lee <jlee@suse.com>
---
 drivers/watchdog/iTCO_wdt.c |   35 +++++++++++++++++++++++++++++++++++
 1 file changed, 35 insertions(+)

--- a/drivers/watchdog/iTCO_wdt.c
+++ b/drivers/watchdog/iTCO_wdt.c
@@ -38,6 +38,7 @@
  *	document number TBD                   : Panther Point
  *	document number TBD                   : Lynx Point
  *	document number TBD                   : Lynx Point-LP
+ *	document number TBD		      : Wellsburg
  */
 
 /*
@@ -130,6 +131,7 @@ enum iTCO_chipsets {
 	TCO_PPT,	/* Panther Point */
 	TCO_LPT,	/* Lynx Point */
 	TCO_LPT_LP,	/* Lynx Point-LP */
+	TCO_WBG,        /* Wellsburg */
 };
 
 static struct {
@@ -195,6 +197,7 @@ static struct {
 	{"Panther Point", 2},
 	{"Lynx Point", 2},
 	{"Lynx Point_LP", 2},
+	{"Wellsburg", 2},
 	{NULL, 0}
 };
 
@@ -377,6 +380,38 @@ static DEFINE_PCI_DEVICE_TABLE(iTCO_wdt_
 	{ PCI_VDEVICE(INTEL, 0x9c45), TCO_LPT_LP},
 	{ PCI_VDEVICE(INTEL, 0x9c46), TCO_LPT_LP},
 	{ PCI_VDEVICE(INTEL, 0x9c47), TCO_LPT_LP},
+	{ PCI_VDEVICE(INTEL, 0x8d40), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d41), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d42), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d43), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d44), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d45), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d46), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d47), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d48), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d49), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d4a), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d4b), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d4c), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d4d), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d4e), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d4f), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d50), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d51), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d52), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d53), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d54), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d55), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d56), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d57), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d58), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d59), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d5a), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d5b), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d5c), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d5d), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d5e), TCO_WBG},
+	{ PCI_VDEVICE(INTEL, 0x8d5f), TCO_WBG},
 	{ 0, },			/* End of list */
 };
 MODULE_DEVICE_TABLE(pci, iTCO_wdt_pci_tbl);
